common/main/cpp/native_c4replicator.cc

```diff
@@ -19,262 +19,174 @@
-        if (localClass == nullptr)
-            return false;
-
-        cls_C4ReplStatus = reinterpret_cast<jclass>(env->NewGlobalRef(localClass));
-        if (cls_C4ReplStatus == nullptr)
-            return false;
-
-        m_C4ReplStatus_init = env->GetMethodID(cls_C4ReplStatus, "<init>", "(IJJJIII)V");
-        if (m_C4ReplStatus_init == nullptr)
-            return false;
-    }
-
-    // C4DocumentEnded, constructor, and fields
-    {
-        jclass localClass = env->FindClass("com/couchbase/lite/internal/core/C4DocumentEnded");
-        if (localClass == nullptr)
-            return false;
-
-        cls_C4DocEnded = reinterpret_cast<jclass>(env->NewGlobalRef(localClass));
-        if (cls_C4DocEnded == nullptr)
-            return false;
-
-        m_C4DocEnded_init = env->GetMethodID(
-                cls_C4DocEnded,
-                "<init>",
-                "(JLjava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;IJIIIZ)V");
-        if (m_C4DocEnded_init == nullptr)
-            return false;
-    }
-
-    {
-        jclass localClass = env->FindClass("com/couchbase/lite/internal/ReplicationCollection");
-        if (localClass == nullptr)
-            return false;
-
-        cls_ReplColl = reinterpret_cast<jclass>(env->NewGlobalRef(localClass));
-        if (cls_ReplColl == nullptr)
-            return false;
-
-        f_ReplColl_token = env->GetFieldID(cls_ReplColl, "token", "J");
-        if (f_ReplColl_token == nullptr)
-            return false;
-
-        f_ReplColl_scope = env->GetFieldID(cls_ReplColl, "scope", "Ljava/lang/String;");
-        if (f_ReplColl_scope == nullptr)
-            return false;
-
-        f_ReplColl_name = env->GetFieldID(cls_ReplColl, "name", "Ljava/lang/String;");
-        if (f_ReplColl_name == nullptr)
-            return false;
-
-        f_ReplColl_options = env->GetFieldID(cls_ReplColl, "options", "[B");
-        if (f_ReplColl_options == nullptr)
-            return false;
-
-        f_ReplColl_pushFilter = env->GetFieldID(
-                cls_ReplColl,
-                "c4PushFilter",
-                "Lcom/couchbase/lite/internal/ReplicationCollection$C4Filter;");
-        if (f_ReplColl_pushFilter == nullptr)
-            return false;
-
-        f_ReplColl_pullFilter = env->GetFieldID(
-                cls_ReplColl,
-                "c4PullFilter",
-                "Lcom/couchbase/lite/internal/ReplicationCollection$C4Filter;");
-        if (f_ReplColl_pullFilter == nullptr)
-            return false;
-
-        m_ReplColl_filterCallback = env->GetStaticMethodID(
-                cls_ReplColl,
-                "filterCallback",
-                "(JLjava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;IJZ)Z");
-        if (m_ReplColl_filterCallback == nullptr)
-            return false;
-    }
-
-    jniLog("replicator initialized");
-    return true;
-}
+namespace litecore::jni {
```

Comment:
This makes the above using namespace redundant but it won't hurt anything

Reply:
Is that true?  The "namespace" block ends at line 333, but the "using namespace" applies to the whole file.  I think I have that right.